tests/scripts/all.sh

```diff
@@ -1886,13 +1925,18 @@ run_component () {
     # Unconditionally create a seedfile that's sufficiently long.
     # Do this before each component, because a previous component may
     # have messed it up or shortened it.
-    dd if=/dev/urandom of=./tests/seedfile bs=64 count=1
+    dd if=/dev/urandom of=./tests/seedfile bs=64 count=1 >/dev/null 2>&1
 
     # Run the component code.
-    "$@"
+    if [ $QUIET -eq 1 ]; then
+        # msg() is silenced, so just print the component name here
+        echo "${current_component#component_}"
+    fi
+    redirect_out "$@"
 
     # Restore the build tree to a clean state.
     cleanup
+    current_component=""
```

Comment:
Since we aren't planning to use the component name anymore, shouldn't this be `unset current_component`?